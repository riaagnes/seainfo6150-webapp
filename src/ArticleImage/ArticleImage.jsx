import React from "react";
import styles from "./ArticleImage.module.css";

const ArticleImage = (props) => {
  return <img className={styles.image} alt={props.title} src={props.url} />;
};

export default ArticleImage;
