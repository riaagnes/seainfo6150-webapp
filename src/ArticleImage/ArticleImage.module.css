img {
    float: top;
    margin-top: .5px;
    border-radius: 50%; 
    border:5px solid  #c00;;
    position: relative;
    height:6em;
    width: 8em; 
}
  