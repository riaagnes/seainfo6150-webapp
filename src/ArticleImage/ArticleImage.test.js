import React from "react";
import ArticleImage from "./ArticleImage";

describe("ArticleImage tests", () => {
    it("renders correctly", () => {
      const { container } = render(<ArticleImage />);
      expect(container).toMatchSnapshot();
    });
});