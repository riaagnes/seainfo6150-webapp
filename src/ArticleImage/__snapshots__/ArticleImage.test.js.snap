// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage tests renders correctly 1`] = `
<div>
  <img
    class="image"
  />
</div>
`;
